catalogs/glade-cesm2-le.json

```diff
@@ -115,5 +115,5 @@
   "esmcat_version": "0.0.1",
   "id": "glade-cesm2-le",
   "description": "ESM collection for the CESM2 LENS data stored on GLADE in /glade/campaign/cgd/cesm/CESM2-LE/timeseries",
-  "last_updated": "2021-12-15T20:25:10+00:00"
+  "last_updated": "2021-01-19T20:25:10+00:00"
```

Comment:
It's 2022 :)

Reply:
Great catch :)

Reply:
Interesting... are you updating this manually or Is this a bug in ecgtools ( I thought ecgtools was updating this field accordingly)?

Reply:
manually... didn't regenerate the entire catalog